src/app/tests/suites/certification/Test_TC_OO_1_1.yaml

```diff
@@ -38,7 +38,7 @@ tests:
       command: "readAttribute"
       attribute: "ClusterRevision"
       response:
-          value: 5
+          value: 6
```

Comment:
I think you're going to get some test failures here because now the cluster revision won't match the cluster revision in the app. If you don't get failures, that means the test isn't running. But I think it is. It should be.

Reply:
Would it need to also get updated elsewhere? This was specifically requested from the issue raised in the description

Reply:
A few things need to happen here. The revision should be 6, that is stated by both spec and test plan. In order for that change to land nicely, we need to get all the examples (zap files) updated to match this cluster rev as well. Leaving it a 5 but still adding in the new feature is not the right solution.

Reply:
This runs against the all cluster app in the CI, so that will need to be updated at the least to change the cluster revision. If the LT feature is supported, the attributes also need to be turned on. And you will need to update the ci-pics-values file to reflect that.

Reply:
Once all-clusters is done, the relevant owners for the other examples will also need to update their code.